packages/react/src/components/Provider/ProviderBox.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { commonPropTypes } from '../../lib'
+import createComponent, { CreateComponentReturnType } from '../../lib/createComponent'
+import { ReactProps } from '../../types'
+import { isFragment } from 'react-is'
+
+const ProviderBox: CreateComponentReturnType<ReactProps<{}>> = createComponent<any>({
+  displayName: 'ProviderBox',
+
+  className: 'ui-provider__box',
+
+  propTypes: {
+    ...commonPropTypes.createCommon(),
+  },
+
+  render(config, props) {
+    const { ElementType, classes, unhandledProps } = config
+    const { children } = props
+    if (isFragment(<ElementType />)) {
```

Comment:
```suggestion
    if (ElementType === React.Fragment) {
```

Do we really need `react-is` there?

Reply:
Thanks! Definitely!